data/projects/l/lifinance.yaml

```diff
@@ -16,70 +17,267 @@ blockchain:
     tags:
       - deployer
       - eoa
-  - address: "0x2fd525b8b2e2a69d054dccb033a0e33e0b4ab370"
+  - address: "0xCB1e9fA11Edd27098A6B2Ff23cB6b79183ccf8Ee"
+    networks:
+      - any_evm
+    tags:
+      - deployer
+      - eoa
+  - address: "0x11F1022cA6AdEF6400e5677528a80d49a069C00c"
+    networks:
+      - any_evm
+    tags:
+      - deployer
+      - eoa
+  - address: "0x552008c0f6870c2f77e5cC1d2eb9bdff03e30Ea0"
+    networks:
+      - any_evm
+    tags:
+      - deployer
+      - eoa
+  - address: "0x4577a46A3eCf44E0ed44410B7793977ffbe22CE0"
+    networks:
+      - any_evm
+    tags:
+      - deployer
+      - eoa
+  - address: "0x29DaCdF7cCaDf4eE67c923b4C22255A4B2494eD7"
```

Comment:
We weren't able to identify contracts deployed from this address. If it's just a project EOA, please remove the `deployer` tag. If it serves as a wallet for moving funds, you can give it a `wallet` tag (optional).